datafusion/core/src/datasource/file_format/parquet.rs

```diff
@@ -1449,8 +1420,14 @@ mod tests {
         // column c1
         let c1_stats = &stats.column_statistics[0];
         assert_eq!(c1_stats.null_count, Precision::Exact(1));
-        assert_eq!(c1_stats.max_value, Precision::Absent);
-        assert_eq!(c1_stats.min_value, Precision::Absent);
```

Comment:
The test behaviour changes due to Utf8 is not supported in

https://github.com/apache/datafusion/blob/ea21b08e477cc48f458917c132f79c4980c957c1/datafusion/core/src/datasource/file_format/parquet.rs#L298-L376

Reply:
It is an improvement@